Clinical-Genomics/scout
# Fix #1494: always show prioritized cases - redux

This PR adds a functionality / fixes a bug.

**How to test**:
1. prioritise a case you can't find on the limit searched (e.g. n>100, or search for something specific)
1.  ensure that the prioritised case is always visible regardless of sort order selection, case searches etc.

**Expected outcome**:
The functionality should be working

**Review:**
